examples/example-ssl.rs

```diff
@@ -108,6 +126,7 @@ mod example {
             opts.optopt("", "ca-cert", "Specify the trusted CA certificates", "FILE");
             opts.optopt("", "client-cert", "Specify the client certificate", "FILE");
             opts.optopt("", "client-key", "Specify key for the client certificate", "FILE");
+            opts.optflag("", "verify-hostname", "Performing server name verification.");
```

Comment:
i would like the program to do verification by default - that's what we generally want to advertise, and only have the option to disable it.